ta/mk/ta_dev_kit.mk

```diff
@@ -20,6 +20,7 @@ link-out-dir := $(out-dir)	# backward compat
 link-out-dir$(sm) := $(out-dir)
 
 user-ta-uuid := $(BINARY)
+user-ta-version := $(if $(CFG_TA_VERSION),$(CFG_TA_VERSION),0)
```

Comment:
`CFG_TA_VERSION` should be documented somewhere.

Reply:
I plan to update documentation for encrypted TAs. So along with that I will document `CFG_TA_VERSION` as well.